cms/envs/common.py

```diff
@@ -1123,6 +1127,14 @@
     'graphical_slider_tool',
 ]
 
+
+################################ Settings for Microsites ################################
+
+### Select an implementation for the microsite backend
+MICROSITE_BACKEND = 'microsite_configuration.backends.filebased.SettingsFileMicrositeBackend'
+MICROSITE_TEMPLATE_BACKEND = 'microsite_configuration.backends.base.BaseMicrositeTemplateBackend'
```

Comment:
I know BaseMicrositeTemplateBackend is not abstract, but it seems odd to reference it as a concrete class with the sort of name that it has.  Does it make sense to create wrapper classes for Base that are more specific to the types of backends they represent?